src/compile.js

```diff
@@ -878,6 +951,11 @@ Compiler.prototype.vseqexpr = function (exprs, data) {
     var ret;
     Sk.asserts.assert(data === undefined || exprs.length === data.length);
     ret = [];
+
+    // if (exprs.length === 1 && exprs[0].constructor === Sk.astnodes.Starred) {
```

Comment:
It seems like these sorts of commented blocks end up staying in the code forever.  Can we either delete such things before merging or put a descriptive comment explaining what this is so that someone in the future can fix and uncomment it (if that's what needs to be done)?